From cccfb9e37a512e8cfb23c93fb4e2961c62597525 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Wed, 29 Jul 2020 12:06:39 +0800
Subject: [PATCH] Test case11 for change FILE_GUID for one module

Description: Change the FILE_GUID of TestPkg/Sec/SecMain.inf.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/Sec/SecMain.inf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/TestPkg/Sec/SecMain.inf b/TestPkg/Sec/SecMain.inf
index 3066e0d102..7700b3595d 100644
--- a/TestPkg/Sec/SecMain.inf
+++ b/TestPkg/Sec/SecMain.inf
@@ -10,7 +10,7 @@
 [Defines]
   INF_VERSION                    = 0x00010005
   BASE_NAME                      = SecMain
-  FILE_GUID                      = df1ccef6-f301-4a63-9661-fc6030dcc880
+  FILE_GUID                      = df1ceef6-f371-4a68-9651-cd6821dec470
   MODULE_TYPE                    = SEC
   VERSION_STRING                 = 1.0
   ENTRY_POINT                    = SecMain
-- 
2.27.0.windows.1

